README.md

```diff
@@ -835,6 +835,9 @@ You can also fork the project on github and run your fork's [build workflow](.gi
                                      interactively
     --ap-list-mso                    List all supported multiple-system
                                      operators
+    --certificate CERTFILE           Path to client certificate file in PEM format. May include the private key.
+    --certificate-key KEYFILE        Path to private key file for client certificate
+    --certificate-key-pass PASSWORD  Password for certificate private key, if encrypted. If not provided and the key is encrypted, yt-dlp will ask interactively
```

Comment:
imo, it should be `--client-certificate`, `--client-certificate-key`, `--client-certificate-password`.

Also, it would be great if password can be obtained from netrc